charts/infra-apps/README.md

```diff
@@ -37,6 +37,13 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | certManager.repoURL | string | [repo](https://charts.jetstack.io) | Repo URL |
 | certManager.targetRevision | string | `"1.1.*"` | [cert-manager Helm chart](https://github.com/jetstack/cert-manager/tree/master/deploy/charts/cert-manager) version |
 | certManager.values | object | [upstream values](https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml) | Helm values |
+| certManagerIssuers.chart | string | `"cert-manager-issuers"` |  |
```

Comment:
something is fishy in the creation of this.

Reply:
Indeed, the comment was wrong in values.yaml.